src/modules/pokemons/PokemonNameType.ts

```diff
@@ -4,1205 +4,1209 @@ To update this type when adding new pokemon:
-    | 'Alcremie (Love Vanilla)'
-    | 'Alcremie (Love Lemon)'
-    | 'Alcremie (Love Mint)'
-    | 'Alcremie (Love Matcha)'
-    | 'Alcremie (Love Ruby Cream)'
-    | 'Alcremie (Love Caramel)'
-    | 'Alcremie (Love Rainbow)'
-    | 'Alcremie (Love Ruby Swirl)'
-    | 'Alcremie (Love Salted)'
-    | 'Alcremie (Star Vanilla)'
-    | 'Alcremie (Star Lemon)'
-    | 'Alcremie (Star Mint)'
-    | 'Alcremie (Star Matcha)'
-    | 'Alcremie (Star Ruby Cream)'
-    | 'Alcremie (Star Caramel)'
-    | 'Alcremie (Star Rainbow)'
-    | 'Alcremie (Star Ruby Swirl)'
-    | 'Alcremie (Star Salted)'
-    | 'Alcremie (Clover Vanilla)'
-    | 'Alcremie (Clover Lemon)'
-    | 'Alcremie (Clover Mint)'
-    | 'Alcremie (Clover Matcha)'
-    | 'Alcremie (Clover Ruby Cream)'
-    | 'Alcremie (Clover Caramel)'
-    | 'Alcremie (Clover Rainbow)'
-    | 'Alcremie (Clover Ruby Swirl)'
-    | 'Alcremie (Clover Salted)'
-    | 'Alcremie (Flower Vanilla)'
-    | 'Alcremie (Flower Lemon)'
-    | 'Alcremie (Flower Mint)'
-    | 'Alcremie (Flower Matcha)'
-    | 'Alcremie (Flower Ruby Cream)'
-    | 'Alcremie (Flower Caramel)'
-    | 'Alcremie (Flower Rainbow)'
-    | 'Alcremie (Flower Ruby Swirl)'
-    | 'Alcremie (Flower Salted)'
-    | 'Alcremie (Ribbon Vanilla)'
-    | 'Alcremie (Ribbon Lemon)'
-    | 'Alcremie (Ribbon Mint)'
-    | 'Alcremie (Ribbon Matcha)'
-    | 'Alcremie (Ribbon Ruby Cream)'
-    | 'Alcremie (Ribbon Caramel)'
-    | 'Alcremie (Ribbon Rainbow)'
-    | 'Alcremie (Ribbon Ruby Swirl)'
-    | 'Alcremie (Ribbon Salted)'
-    | 'Falinks'
-    | 'Pincurchin'
-    | 'Snom'
-    | 'Frosmoth'
-    | 'Stonjourner'
-    | 'Eiscue'
-    | 'Eiscue (noice)'
-    | 'Indeedee'
-    | 'Morpeko'
-    | 'Morpeko (hangry)'
-    | 'Cufant'
-    | 'Copperajah'
-    | 'Dracozolt'
-    | 'Arctozolt'
-    | 'Dracovish'
-    | 'Arctovish'
-    | 'Duraludon'
-    | 'Dreepy'
-    | 'Drakloak'
-    | 'Dragapult'
-    | 'Zacian (Battle Hero)'
-    | 'Zacian (Crowned Sword)'
-    | 'Zamazenta (Battle Hero)'
-    | 'Zamazenta (Crowned Shield)'
-    | 'Eternatus'
-    | 'Kubfu'
-    | 'Urshifu (Single Strike)'
-    | 'Urshifu (Rapid Strike)'
-    | 'Zarude'
-    | 'Regieleki'
-    | 'Regidrago'
-    | 'Glastrier'
-    | 'Spectrier'
-    | 'Calyrex'
-    | 'Ice Rider Calyrex'
-    | 'Shadow Rider Calyrex'
-    | 'MissingNo.'
-    | 'Flying Pikachu'
-    | 'Surfing Pikachu'
-    | 'Armored Mewtwo'
-    | 'Santa Snorlax'
-    | 'Spooky Togepi'
-    | 'Surprise Togepi'
-    | 'Spooky Bulbasaur'
-    | 'Pikachu (Gengar)'
-    | 'Let\'s Go Pikachu'
-    | 'Let\'s Go Eevee'
-    | 'Bulbasaur (clone)'
-    | 'Ivysaur (clone)'
-    | 'Venusaur (clone)'
-    | 'Charmander (clone)'
-    | 'Charmeleon (clone)'
-    | 'Charizard (clone)'
-    | 'Squirtle (clone)'
-    | 'Wartortle (clone)'
-    | 'Blastoise (clone)'
-    | 'Mega Venusaur'
-    | 'Mega Charizard X'
-    | 'Mega Gyarados'
-    | 'Mega Gardevoir'
-    | 'Mega Rayquaza'
-    | '???';
+export const pokemonNameList = ['Bulbasaur',
```

Comment:
I changed this to allow for a safe type guard on `PokemonNameType` using the pattern found here: https://stackoverflow.com/questions/36836011/checking-validity-of-string-literal-union-type-at-runtime

If this is an unwanted change, let me know, and I can go back to using the unsafe type guard (which just checked if it wasn't an `EggNameType` or `FossilNameType`).